### PR TITLE
COL-14 added Class Player

### PR DESCRIPTION
**Pull request by JuliaTrzebuchowska**
Added Class Player to manage players and their attributes. Small adjustments will be made after the creation of associated classes.

### CODE REVIEW

File: core/src/model/views/Player.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package model.views;
+
+import java.util.List;
+
+public class Player {
+    private int id;
```
Comment (WiktoriaKrzyzanska) [inline core/src/model/views/Player.java:6]: I think it can be made final

File: core/src/model/views/Player.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    private String name;
```
Comment (WiktoriaKrzyzanska) [inline core/src/model/views/Player.java:7]: This to can be made final

**Review by Ikar17 — APPROVED** [review]
Good job

File: core/src/model/views/Player.java (hunk 1 above)
Comment (JuliaTrzebuchowska) [reply core/src/model/views/Player.java:6]: this will result in error
cause: id field has a setter

**Pull request merged**